index.css

```diff
@@ -0,0 +1,313 @@
+    color: #000;
+    border: 0;
+    border-bottom: 2px solid #ccc;;
+    display: block;
+    width: 100%;
+    padding: 15px 0;
+    font-size: 18px;
+}
+.form-input input:focus,
+.form-input input:valid,
+.form-input textarea:focus,
+.form-input textarea:valid {
+    outline: 0
+}
+
+.btn {
+    background-color: #F64C72;
+    cursor: pointer;
+    display: inline-block;
+    width: 50%;
+    padding: 15px 0;
+    
+    font-family: inherit;
+    font-size: 16px;
+    border: 0;
+    border-radius: 5px;
+}
+.btn:focus {
+    outline: 0;
+}
+.btn:active {
+    transform: scale(.98);
+}
+.text {
+    color: #000;
+    margin-top: 30px;
+}
+.form-input label {
+    position: absolute;
+    top: 15px;
+    left: 0;
+}
+.contact-message {
+    vertical-align: middle;
+}
+footer {
+    background-color: #242582
+}
+@media (max-width: 1100) {
+    .profile-link {
+        flex-basis: 20%;
+    }
+}
+
+@media (max-width: 800px){
+    
+    section, .grid-item {
+        padding: 0;
+    }
+    header {
+        font-size: 1em;
+    }
+    .title {
+        font-size: 6em;
+    }
+    .title, .bio {
+        padding: 1%;
+        width: 80%;
+        margin: 0 auto;
+    }
+    .profile-link {
+        flex-basis: 50%;
+    }
+ 
+    .profile-links {
+        grid-template-columns: repeat(2, auto);
+    }
+    .portfolio {
+        /* grid-gap: 2%; */
+        padding: 5%;
+    }
+    .portfolio-item {
+        flex-basis: 50%;
+    }
+    /* .portfolio-item,
+    .portfilio-item::after,
+    .portfolio-item::before {
+        flex-basis: 50%;
+    } */
+    .contact{
+        grid-template-columns: 1fr;
+        flex-direction: column-reverse;    }
+    form {
+        margin: 0 auto;
+    }
+    .contact-message{
+        order: -2;
+    }
+}
+@media (max-width: 500px){
+    * {
+        margin: 0;
+        padding: 0;
+        font-size: 1em;
+    }
+    .container {
+        justify-content: center;
+        width:100%;
+        display: flex;
+        flex-direction: column;
+    }
+    .grid-item {
+        justify-content: center;
+        border: none;
+        flex-basis: 100%;
+        padding: 0;
+        margin: 0;
+    }
+    header {
+        justify-content: center;
+    }
+    .profile-links {
+        grid-template-columns: 1fr;
+        grid-gap: 10px;
+        align-items: center;
+        justify-content: center;
+        padding: 0;
+    }
+    .profile-link {
+        border: none;
+        flex-basis: 100%;
+        margin: 0;
+        padding: 0;
+    }
+    .title {
+        font-size: 5em;
+        margin: 0 auto
+    }
+    .bio {
+        margin: 0 auto;
+    }
+    .portfolio {
+        padding-bottom: 5%;
+    }
+    .portfolio-item {
+        margin: 5%;
+    }
+    .contact{
+        padding: 5%;
+    }
+    form {
+        border: 10px dotted #1b1d8a;
+        margin: 0 auto;
+        padding: 5% 10%;
+    }
+    .contact-message {
+        padding-bottom: 10%;
+    }
+    footer {
+        width: 100vw;
+    }
+}
```

Comment:
My recommendation would be for you to split your CSS file into sections instead of doing it in just one file. It's more readable this way.